rest/v9/guild.ts

```diff
@@ -238,9 +238,9 @@ export type RESTPatchAPIGuildJSONBody = AddUndefinedToPossiblyUndefinedPropertie
 	 */
 	afk_channel_id?: Snowflake | null;
 	/**
-	 * afk timeout in seconds, can be set to: `60`, `300`, `900`, `1800`, `3600`
```

Comment:
keep the comment

Reply:
Only in `RESTPatchAPIGuildJSONBody` or also in `RESTPostAPIGuildsJSONBody`?

Reply:
Everywhere it was removed

Reply:
Oh, ok